internal/jennies/java/rawtypes.go

```diff
@@ -120,6 +120,9 @@ func (jenny RawTypes) formatEnum(pkg string, object ast.Object) ([]byte, error)
 	enum := object.Type.AsEnum()
 	values := make([]EnumValue, len(enum.Values))
 	for i, value := range enum.Values {
+		if value.Name == "" {
+			value.Name = "None"
```

Comment:
Isn't "None" a Python concept?

Reply:
Maybe... but the name that we are using in Golang is "None" and I used the same one.